module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <utility>
+#include <vector>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>&);
```

Comment:
You should consider aliases for longer names like std::pair<uint8_t, uint8_t>.

Reply:
not a scope of the offer, corrected on past branch